### ISSUE TITLE
provide screen-reader-accessible labels for links with the same names

### ISSUE THREAD

**Issue by jgiardino** [cloudmosaic/quickstarts#44]
There are two guiding principles to follow for providing a11y labels:

1. Whenever links or buttons with the same visible label appear in the UI but have different behaviors, they should have some additional context provided for screen reader users. This allows screen reader users to scan a list of links or buttons on a page and know what their expected behavior is without having to scan the full contents of the page.

2. Whenever links or buttons have visible text (i.e as opposed to an icon as the visible "label"), this visible text should also be the text that is announced first before any additional text provided for screen reader users. Because sometimes screen reader users can see the screen (e.g. low vision, users with dyslexia). Therefore the text they click on is the text they expect to hear first, to avoid confusion. 

This means that if a link is displayed as "View documentation" multiple times, then the the heading that provides context for each link should be included in the text that's announced for that link. For example "View documentation, CLI command reference".

There are several ways to implement this behavior. Here are a couple.

1. Link has `aria-labelledby="[id of link] [id of heading]"
2. Link includes the heading text in a span with the [pf screen reader only class](https://www.patternfly.org/v4/utilities/accessibility#screen-reader-only): `<a ...>View documentation <span class="pf-u-screen-reader">[heading text]</span>`
